### PR TITLE
[fuzzing] Add hook to automatically update gridl grammar

### PR DESCRIPTION
**Pull request by pyoor**
This hook is already running on community-tc.

### CODE REVIEW

File: config/projects/fuzzing.yml [hunk 1]
```diff
@@ -401,6 +401,46 @@ fuzzing:
...
+          $fromNow: 6 hours
+        metadata:
+          name: gr-idl-update
+          owner: jkratzer@mozilla.com
+          source: https://github.com/MozillaSecurity/gridl
+          description: Hook for updating gr.idl grammar
+        workerType: ci
+        provisionerId: proj-fuzzing
+      schedule:
+        - "0 8 * * 1"
```
Comment (jwhitlock) [inline config/projects/fuzzing.yml:443]: The [hook on community-tc](https://community-tc.services.mozilla.com/hooks/project-fuzzing/gr-idl-update) is currently not scheduled. Do you want me to apply this change and schedule it?

**Review by jwhitlock — APPROVED** [review]
Thanks @pyoor!

File: config/projects/fuzzing.yml (hunk 1 above)
Comment (pyoor) [reply config/projects/fuzzing.yml:443]: Sure thing.  Thanks @jwhitlock!

Comment (jwhitlock) [reply config/projects/fuzzing.yml:443]: 🚀 scheduled!

**Pull request merged**